src/connect/components/tree-view-item/tree-view-item.tsx

```diff
@@ -37,26 +41,52 @@ export enum ItemStatus {
     Offline = 'offline',
 }
 
-export interface TreeItem {
+export interface TreeItem<T extends string = string> {
     id: string;
     label: string;
     type: ItemType;
     status?: ItemStatus;
     expanded?: boolean;
-    children?: TreeItem[];
+    children?: TreeItem<T>[];
     isSelected?: boolean;
-    options?: ConnectDropdownMenuProps['items'];
+    options?: ConnectDropdownMenuItem<T>[];
 }
 
-export interface ConnectTreeViewItemProps
+export const DefaultOptions = [
+    {
+        id: 'duplicate',
+        label: 'Duplicate',
```

Comment:
My only concern is that I think this library should not know about translations. Maybe this is an small use case, and we can skip the rule. what do you think?

Reply:
I think here we can consider it a default label, and it's on a Connect component so it is a bit more specific.